README.md

```diff
@@ -65,37 +81,48 @@ CKEDITOR.replace('editor', {
 })
 ```
 
-## Configuration
+## <a id="config" href="#config"></a> Configuration
+
+### <a id="config-plugin" href="#config-plugin"></a> Plugin configuration
 
-Initialize a plugin with additional options:
+To apply a custom configuration, initialize the plugin providing additional
+options:
 
 ```javascript
-UPLOADCARE_LOCALE = 'ru' // set locale if you wish
+UPLOADCARE_LOCALE = 'ru' // set a preferred locale if needed
 
 CKEDITOR.replace('editor', {
   extraPlugins: 'uploadcare',
   toolbar: [['Uploadcare', /* Your toolbar items */]],
   uploadcare: {
-    publicKey: 'YOUR_PUBLIC_KEY',
-    multiple: true,
-    crop: '1:1,4:3',
+    publicKey: 'YOUR_PUBLIC_KEY', // set your public API key here
+    multiple: true, // allow multi-file uploads
+    crop: '1:1,4:3', // set crop options when handling images
```

Comment:
Also, users can add more "object key" options. Can we say about it here in `/* comment */` or after this code?